src/lib/utilities/route-for.ts

```diff
@@ -40,8 +40,9 @@ export type EventHistoryParameters = Pick<
   RouteParameters,
   'namespace' | 'workflow' | 'run' | 'view' | 'queryParams'
 >;
-export type EventParameters = Required<
-  Pick<RouteParameters, 'namespace' | 'workflow' | 'run' | 'view' | 'eventId'>
+export type EventParameters = Pick<
```

Comment:
@Alex-Tideman this is how I was able to fix the type errors. I'm worried that this may break something in the future. Was their a better way of handling this?

Reply:
Yup it's okay to remove that. One good way to check is to see where EventParameters is used throughout the app with a global search in your IDE. It was only used in the test file, so I updated the tests to not include view.